### PR TITLE
added Laravel 8 compatible Factory and Model generation

### PR DESCRIPTION
**Pull request by Rezrazi**
changed the `module:make-factory` argument description from `ModelFactory` to `Model` that way the command generates a factory with a model name rather that a factory name as argument

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,5 @@ build
 coverage
 .php_cs.cache
 .phpunit.result.cache
+
+.idea
```
Comment (nWidart) [inline .gitignore:10]: Can you remove this please?

File: src/Commands/stubs/model.stub [hunk 2]
```diff
@@ -3,8 +3,16 @@
 namespace $NAMESPACE$;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
 
 class $CLASS$ extends Model
 {
+    use HasFactory;
+
+    protected static function newFactory()
```
Comment (nWidart) [inline src/Commands/stubs/model.stub:12]: I think this should probably below the fillable properties, to keep properties on top.

**Review by nWidart — COMMENTED** [review]
Thanks for the PR!

**Review by Rezrazi — COMMENTED** [review]
applied the following changes :
- updated `.gitignore` to remove IDE specific lines
- updated `model.stub` to fix the method order

**Review by Rezrazi — COMMENTED** [review]
updated the snapshots to get the `ModelMakeCommandTest` tests to pass

**Comment by nWidart** [comment]
Looks great thanks for the changes!

**Pull request merged**